client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -291,7 +293,10 @@ const Checkout = React.createClass( {
 			} );
 		}
 
-		if ( cart.create_new_blog && receipt && isEmpty( receipt.failed_purchases ) ) {
+		if (
+			( cart.create_new_blog && receipt && isEmpty( receipt.failed_purchases ) ) ||
+			( isDomainOnly && cartItems.hasPlan( cart ) )
+		) {
 			notices.info(
 				this.translate( 'Almost done…' )
 			);
```

Comment:
Ah, I see. Very good point. :) I can't test this because I never manage to reproduce the bug. Yeah, I can use the current site, because at this point there's a site created, and we can get the slug from redux state. 👍